perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -950,7 +950,9 @@ sub admin_templates_external_status_code_hook {
     my $task_type = $c->get_param('task_type') || '';
     my $task_state = $c->get_param('task_state') || '';
 
-    return "$res_code,$task_type,$task_state";
+    my $code = "$res_code,$task_type,$task_state";
+    $code = '' if $code eq ',,';
```

Comment:
I assume there's no chance on only one of these being blank?

Reply:
As long as the resolution code is set, the looking code in Bromley.pm looks for any of "RES,TYPE,STATE", "RES,TYPE,", "RES,,STATE", "RES,,". And the waste update code sets the code to "RES,,". So I think it's okay, though I get confused.